docs/incident_response.md

```diff
@@ -112,15 +103,10 @@ metadave> we'll followup with an incident report                    5:00 PM
 
 Resolve the incident in pagerduty. Once an incident is resolved, no additional notifications are sent and the incident cannot be triggered again.
 
-## Incident report
```

Comment:
I think we should continue incident reports in the MozIT cluster. I like Mana to index them, because they are available to staff but not public. I'd like to update the target rather than deleting this section.

Reply:
Thanks for the feedback @jwhitlock. I'll create a new page on Mana and update the link.

Reply:
As we discussed briefly in IRC, I'll keep the current link (within the MozMEAO hierarchy) for now.